src/main/java/io/vertx/redis/RedisClient.java

```diff
@@ -1833,7 +1834,20 @@ static RedisClient create(Vertx vertx, RedisOptions config) {
    * group: string
    */
   @Fluent
-  RedisClient setBinary(String key, String value, Handler<AsyncResult<Void>> handler);
+  RedisClient setBinary(String key, Buffer value, Handler<AsyncResult<Void>> handler);
+
+  /**
+   * Set the string value of a key
+   *
+   * @param key     Key of which value to set
+   * @param value   New value for the key
+   * @param options Set options
+   * @param handler Handler for the result of this call.
+   * @since 1.0.0
```

Comment:
oh, well, I think it's a valuable information. Maybe instead of just @since, add something like @since Redis 1.0.0 to indicate it refers to Redis version and not the version of the current API.